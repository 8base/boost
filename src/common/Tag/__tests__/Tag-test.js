import React from 'react';
import renderer from 'react-test-renderer';
import { Tag } from '../';

it('As developer, I can use tag for collect common props.', () => {
  const tree = renderer.create(<Tag tagName="div" onClick={ () => null }>children</Tag>);

  expect(tree.toJSON()).toMatchSnapshot();
});

it('As developer, I can pass data attributes to the tag.', () => {
  const tree = renderer.create(<Tag tagName="div" data-attr={ 10 } />);

  expect(tree.toJSON()).toMatchSnapshot();
});
