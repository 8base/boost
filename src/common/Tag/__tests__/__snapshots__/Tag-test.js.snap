// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`As developer, I can pass data attributes to the tag. 1`] = `
<div
  data-attr={10}
/>
`;

exports[`As developer, I can use tag for collect common props. 1`] = `
<div
  onClick={[Function]}
>
  children
</div>
`;
